rpm/fs/etc/rc.d/init.d/mad

```diff
@@ -17,15 +17,34 @@ user="mad"
 prog="mad"
 dir="/opt/mad"
 log_dir="$dir/logs"
-config="/opt/mad/config/config.json"
-logging_config="-Dlogback.configurationFile=/opt/mad/config/logback.xml"
 exec="/opt/mad/bin/mad"
 pid_file="/var/run/mad/mad.pid"
+export CONFIG_FILE="/opt/mad/config/config.hocon"
 export JAVA_HOME="$(dirname $(dirname $(readlink -f $(which java))))"
-export JAVA_OPTS="$logging_config"
+export JVM_XMS="64m"
+export JVM_XMX="1024m"
+export LOGGING_CONFIG_FILE="/opt/mad/config/logback.xml"
 
 [ -e /etc/sysconfig/$prog ] && . /etc/sysconfig/$prog
 
+logging_config="-Dlogback.configurationFile=${LOGGING_CONFIG_FILE}"
+if [ -z "$JAVA_OPTS" ]; then
```

Comment:
Just curious; do we (by which I mean you) have a use case where you override JAVA_OPTS?